src/GovUk.Education.ExploreEducationStatistics.Admin/Services/PublicationService.cs

```diff
@@ -119,6 +119,11 @@ public class PublicationService : IPublicationService
                         return await ValidateSelectedTopic(updatedPublication.TopicId);
                     }
 
+                    if (publication.Title != updatedPublication.Title)
+                    {
+                        return await _userService.CheckCanUpdatePublicationTitle();
```

Comment:
Looks like `PublicationService` `UpdatePublication` is completely untested, so this would have been a good opportunity to introduce some tests for it although appreciate it's not an easy method to write tests for and this change would have no affect on the test had it existed since we usually use a `userService` mock that always returns true.

Reply:
There were some unit tests for UpdatePublication, but I added a couple that use a `userService` mock.